tests/fixtures/expected_diff.py

```diff
@@ -1,8 +1,44 @@
 EXPECTED_DIFF = """{
```

Comment:
let's keep old diffs too :) just make a new constant `NESTED_DIFF` and rename the old one to `PLAIN_DIFF`